frontend/src/component/personalDashboard/MyProjects.tsx

```diff
@@ -137,8 +137,12 @@ export const MyProjects: FC<{
                 </List>
             </SpacedGridItem>
             <SpacedGridItem item lg={4} md={1}>
-                {activeProjectStage === 'onboarded' ? (
-                    <ProjectSetupComplete project={activeProject} />
+                {activeProjectStage === 'onboarded' &&
+                personalDashboardProjectDetails ? (
```

Comment:
question: can you have an active project stage without personaldashboardprojectdetails? is this redundant?

Reply:
no, but the orval type doesn't allow me to make this state more explicit so I ended up adding two expressions in here